backend/src/utils/array.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Add all {@link elements} to {@link array}. This mutates {@link array} and returns it.
+ * @param array The array to append to
+ * @param elements The elements to append
+ * @returns The appended array
+ */
+export function pushAll<T>(array: T[], elements: T[]): T[] {
+    elements.forEach((element) => array.push(element));
```

Comment:
Add a comment about `.push(...Array(300000).fill(1))` causing a memory error (because all elements are loaded to the stack).